server/routerlicious/packages/lambdas/src/scriptorium/lambda.ts

```diff
@@ -88,12 +97,15 @@ export class ScriptoriumLambda implements IPartitionLambda {
         Promise.all(allProcessed).then(
             () => {
                 this.current.clear();
+                metric?.setProperty("processingComplete", true);
                 // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
                 this.context.checkpoint(batchOffset!);
+                metric?.setProperty("checkpointComplete", true);
+                metric?.success(`Scriptorium completed processing and checkpointing of batch with offset ${batchOffset?.offset}`);
                 this.sendPending();
             },
             (error) => {
-                Lumberjack.error("An error occured in scriptorium, going to restart", {}, error);
+                metric?.error(`Scriptorium failed to process/checkpoint batch with offset ${batchOffset?.offset}, going to restart`, error);
```

Comment:
Maybe consider to put `${batchOffset?.offset}` into a property instead? which will simplify the error message and will help you to analysis/report logs easier in the future.

Reply:
We are also storing it as a property during metric creation at line 80 above.